### PR TITLE
Internal: Adjust kit library to display kit tier on hover [ED-16127]

### PR DESCRIPTION
**Pull request by ronenboxer**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by davseve — DISMISSED** [review]

File: app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js [hunk 1]
```diff
@@ -23,10 +25,8 @@ export default function useKitCallToAction( kitAccessTier ) {
 	// SubscriptionPlan can be null when the context is not filled (can be happened when using back button in the browser.)
 	const subscriptionPlan = useMemo( () => settings.subscription_plans?.[ kitAccessTier ], [ settings, kitAccessTier ] );
 
-	// Free user should see a generic "Pro" badge.
-	const badgeLabel = userAccessTier === TIERS.free
-		? settings.subscription_plans?.essential.label
-		: subscriptionPlan?.label;
+	subscriptionPlan.label = PromotionChipText[ tierKey ];
+	subscriptionPlan.color = TIERS.free === kitAccessTier ? '#467FF7' : '#92003B';
```
Comment (baghdasarovelementor) [inline app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js:29]: Maybe have const variables for color codes?

File: app/modules/kit-library/assets/js/models/taxonomy-transformer.js [hunk 2]
```diff
@@ -29,14 +29,20 @@ const TaxonomyTransformMap = {
...
 	[ TIERS.essential ]: ESSENTIAL,
 	[ TIERS[ 'essential-oct2023' ] ]: ADVANCED,
 	[ TIERS.expert ]: ADVANCED,
 	[ TIERS.agency ]: ADVANCED,
 };
 
+export const PromotionChipText = {
+	[ FREE ]: __( 'Free', 'elementor ' ),
+	[ ESSENTIAL ]: __( 'Essential', 'elementor ' ),
+	[ ADVANCED ]: __( 'Advanced', 'elementor ' ),
```
Comment (davseve) [inline app/modules/kit-library/assets/js/models/taxonomy-transformer.js:43]:
```suggestion
	[ FREE ]: __( 'Free', 'elementor' ),
	[ ESSENTIAL ]: __( 'Essential', 'elementor' ),
	[ ADVANCED ]: __( 'Advanced', 'elementor' ),
```

File: app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js (hunk 1 above)
Comment (ronenelementor) [reply app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js:29]: I put them in the scss file and added classes, WDYT?

File: app/modules/kit-library/assets/js/models/taxonomy-transformer.js (hunk 2 above)
Comment (ronenelementor) [reply app/modules/kit-library/assets/js/models/taxonomy-transformer.js:43]: Nice, done

**Review by davseve — APPROVED** [review]

**Review by baghdasarovelementor — APPROVED** [review]

**Pull request merged**